testsuite/classlibrary/TestLcmGcd.sc

```diff
@@ -0,0 +1,156 @@
+
+TestLcmGcd : UnitTest {
+
+
+	// lattice theoretic laws
+	// https://en.wikipedia.org/wiki/Least_common_multiple#Lattice-theoretic
+
+	callTest_commutative_lcm { |a, b|
+		var x = lcm(a, b);
+		var y = lcm(b, a);
+		this.assertEquals(x, y, "lcm(%, %) = lcm(%, %) should be valid"
```

Comment:
What about setting `UnitTest.reportPasses = false` ?

restructuring the code makes it rather unwieldy, or at least I have no idea of how to make a lightweight change here.